src/main/java/com/cinema/model/Orders.java

```diff
@@ -0,0 +1,96 @@
+package com.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+
+@Entity
+public class Orders {
```

Comment:
I would, personally, rename it to "Order" and add an annotation 
@Table(name = "orders")